src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -36,7 +35,7 @@
                                 {:permissions [:post-notifications]
                                  :on-allowed  #(log/info "push notification permissions were allowed")
                                  :on-denied   #(log/error "user denied push notification permissions")}])
-                              (rf/dispatch [:push-notifications/switch true platform/ios?])
+                              (rf/dispatch [:push-notifications/switch true])
```

Comment:
https://github.com/status-im/status-mobile/blob/5ee5d705675f5f0378124311cc69fd1501612b8e/src/legacy/status_im/ui/screens/notifications_settings/events.cljs#L50-L51

`platform/ios?` is redundant